samples/matter/manufacturer_specific/README.rst

```diff
@@ -0,0 +1,276 @@
+.. _matter_manufacturer_specific_sample:
+
+Matter: Manufacturer-specific
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates an implementation of custom manufacturer-specific clusters used by application layer.
+This sample utilizes uses development kit's buttons and LEDs to demonstrate the functionality of the custom ``NordicDevkit`` cluster.
+The device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power, 802.15.4 Thread network.
+You can use this sample as a reference for creating your own application.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+For testing purposes, that is to commission the device and :ref:`control it remotely <matter_manufacturer_specific_network_mode>` through a Thread network, you also need a Matter controller device :ref:`configured on PC or smartphone <ug_matter_configuring>`.
+This requires additional hardware depending on the setup you choose.
+
+.. note::
+    |matter_gn_required_note|
+
+Overview
+********
+.. tabs::
```

Comment:
Just a question. Aren't most of these things pretty much explained already in the User interface section?

Reply:
That's correct, but I'd say that overview is the first thing I'd look at as customer. That's why I wanted to explain thoroughly what the sample does (it mainly utilizes buttons and leds, but also adds devkitname attribute which is available only as an attribute